docs/editing.md

```diff
@@ -23,9 +23,19 @@ The EiC assigns an editor (or a volunteering editor self-assigns) with the comma
 .. note:: If a paper is submitted without a recommended editor, it will show up in the weekly digest email under the category ‘Papers currently without an editor.’ Please review this weekly email and volunteer to edit papers that look to be in your domain. If you choose to be an editor in the issue thread type the command ``@whedon assign @yourhandle as editor``
 ```
 
+### How papers are assigned to editors
+
+By default, unless an editor volunteers, one of the Editors-in-chief (EiCs) will attempt to assign an incoming paper to the most suitable handling editor. While EiCs will make every effort to match a submission with the most appropriate editor, there are a number of situations where an EiC may assign a paper to an editor that doesn't fit entirely within the editor's research domains:
```

Comment:
By default, unless an editor volunteers, the Associated Editor-in-chief (AEiC) on duty will attempt to assign an incoming paper to the most suitable handling editor. While AEiCs will make every effort to match a submission with the most appropriate editor, there are a number of situations where an AEiC may assign a paper to an editor that doesn't fit entirely within the editor's research domains: